packages/zoe/src/typeGuards.js

```diff
@@ -224,7 +224,7 @@ export const InstanceStorageManagerIKit = harden({
       InstanceHandleShape,
       M.remotable('instanceAdmin'),
     ).returns(M.promise()),
-    deleteInstanceAdmin: M.call(InstanceAdminI).returns(),
+    deleteInstanceAdmin: M.call(InstanceAdminShape).returns(),
```

Comment:
This bug was only caught by seeing https://github.com/Agoric/agoric-sdk/pull/8340 fail , because it uses the stricter https://github.com/endojs/endo/pull/1712

Reply:
See https://github.com/Agoric/agoric-sdk/issues/8341